src/mbgl/util/tile_cover.cpp

```diff
@@ -124,19 +124,10 @@ std::vector<TileID> tileCover(const TileCoordinate& tl,
     scanTriangle(tl, tr, br, 0, tiles, scanLine);
     scanTriangle(br, bl, tl, 0, tiles, scanLine);
 
-    t.sort();
-    t.unique();
-
-    t.sort([&](const TileID& a, const TileID& b) {
```

Comment:
Spiral is maybe was a poor choice of words, but tiles currently get loaded from the center to the corners.

It is very noticeable when you change to a satellite style with the window maximized. IMO this is a very nice thing to have because it starts loading exactly where the user is looking at.

![screenshot from 2016-04-26 17-26-11](https://cloud.githubusercontent.com/assets/996307/14821617/22e8ab7c-0bd4-11e6-93af-37d3dded32c7.png)